src/components/checkbox/checkbox.scss

```diff
@@ -12,6 +12,12 @@ $checkbox-top: 12px !default;
   }
 }
 
+// Needed for check to display correctly in IE11
```

Comment:
I don't like the comment here, please explain more about the intention here.

Like it is not showing with:
- Tables with Border Collapse
- Tables with Border Spacing